letters/modernized/16-letters.adoc

```diff
@@ -603,14 +631,19 @@ The time I am limited to, by the vessel`'s sailing this afternoon,
 obliges me to draw to a conclusion, and add no more but my tender love and regard to you,
 and all my dear family, remaining in true gospel love, Your real and affectionate friend,
```

Comment:
This one... where she has the capital "Your real and affectionate friend," I would bump down and make a SSCL 👍🏻